src/main/java/net/neoforged/neoforge/network/filters/GenericPacketSplitter.java

```diff
@@ -208,7 +201,7 @@ public enum RemoteCompatibility {
     }
 
     public static RemoteCompatibility getRemoteCompatibility(Connection manager) {
-        return NetworkRegistry.getInstance().isConnected(manager, ConnectionPhase.ANY, SplitPacketPayload.ID) ? RemoteCompatibility.PRESENT : RemoteCompatibility.ABSENT;
+        return NetworkRegistry.hasChannel(manager, null, SplitPacketPayload.ID) ? RemoteCompatibility.PRESENT : RemoteCompatibility.ABSENT;
```

Comment:
@marchermans was this usage of `ConnectionPhase.ANY` even correct before this change?  I'm thinking it should have been referencing `GenericPacketSplitter#connectionType` instead.